blackmarket/Application/blackmarketApp.swift

```diff
@@ -9,9 +9,19 @@ import SwiftUI
 
 @main
 struct blackmarketApp: App {
+    
+    @State var isAuthenticated = SessionManager.IsAuthenticated()
+
     var body: some Scene {
         WindowGroup {
-            SignInView()
+            Group {
+                isAuthenticated ?
+                                  AnyView(TabBarView()) :
+                                  AnyView(SignInView())
```

Comment:
Can the Group use the views directly  in the body?

```suggestion
                isAuthenticated ? TabBarView() : SignInView()
```